main.cpp

```diff
@@ -371,6 +382,7 @@ extern "C" int GetIPList(void* data, char* requestedHostname, addr_t* addr, int
 
     uint64_t requestedFlags = 0;
     int hostlen = strlen(requestedHostname);
+    // printf("requested hostname: %s\n", requestedHostname);
```

Comment:
to be removed?

Reply:
This code uses commented out printfs all over the place to leave debugging code around that should not be part of the normal output.

Reply:
I agree, the code use a lot of `printf`. It's just that I rather don't add new occurrences. That said we could take care of it once we move everything in the BU codebase.